_locales/_untranslated_en.json

```diff
@@ -1,4 +1,7 @@
 {
+  "menu_copy_link_to_clipboard": {
+    "message": "Copy Link"
```

Comment:
link should be small

Reply:
Yes @Jikstra. This should be "Copy link". You're welcome, but it would be quicker to fix it than to say it. But I don't know what I have to do to be with you. Unfortunately, I can't convince you.

Reply:
telegram has it uppercase but whatever I make it small/